_maps/configs/skipper.json

```diff
@@ -1,8 +1,17 @@
 {
 	"map_name": "Skipper-class Heavy Cruiser",
 	"map_short_name": "Skipper-class",
-	"map_path": "_maps/shuttles/shiptest/ntsv_skipper.dmm",
-	"map_id": "ntsv_skipper",
+	"prefix": "NTSV",
+	"namelists": [
+		"GENERAL",
+		"SPACE",
+		"BRITISH_NAVY",
+		"MYTHOLOGICAL",
+		"WEAPONS",
+		"MERCANTILE"
+	],
+	"map_path": "_maps/shuttles/shiptest/nanotrasen_skipper.dmm",
+	"map_id": "nanotrasen_skipper",
 	"roundstart": true,
 	"job_slots": {
 		"Captain": {
```

Comment:
Given the presence of the Requests Office, which is locked to HoP access, you might want to add a First Officer to the ship. The Skipper's bridge actually is big enough to accomodate two officers and the IAA has a separate office, after all.

Reply:
but it does have a first officer